frontend/src/scenes/error-tracking/errorTrackingIssueSceneLogic.ts

```diff
@@ -132,46 +97,106 @@ export const errorTrackingIssueSceneLogic = kea<errorTrackingIssueSceneLogicType
         ],
 
         eventsQuery: [
-            (s) => [s.issue, s.filterTestAccounts, s.filterGroup, s.dateRange],
-            (issue, filterTestAccounts, filterGroup, dateRange) =>
+            (s) => [(_, props) => props.id, s.filterTestAccounts, s.filterGroup, s.dateRange],
+            (issueId, filterTestAccounts, filterGroup, dateRange) =>
                 errorTrackingIssueEventsQuery({
-                    issue,
+                    issueId,
                     filterTestAccounts: filterTestAccounts,
                     filterGroup: filterGroup,
-                    dateRange,
+                    dateRange: resolveDateRange(dateRange).toDateRange(),
                 }),
         ],
 
-        issueProperties: [
+        issueDateRange: [(s) => [s.issue], (issue) => (issue ? getIssueDateRange(issue) : {})],
+
+        firstSeen: [
             (s) => [s.issue],
-            (issue): Record<string, any> => (issue && issue.earliest ? JSON.parse(issue.earliest) : {}),
+            (issue: ErrorTrackingRelationalIssue | null) => (issue ? dayjs(issue.first_seen) : null),
         ],
+
+        lastSeen: [(s) => [s.summary], (summary: ErrorTrackingIssueSummary | null) => summary?.lastSeen],
+        aggregations: [(s) => [s.summary], (summary: ErrorTrackingIssueSummary | null) => summary?.aggregations],
     }),
 
-    listeners(({ values, actions }) => {
-        const loadIssue = (): void => {
-            if (!values.issueLoading) {
-                const issue = values.issue
-                if (!issue) {
-                    actions.loadRelationalIssue()
-                } else {
-                    actions.loadClickHouseIssue(dayjs(issue.first_seen))
-                }
-            }
-        }
+    loaders(({ props }) => ({
+        issue: {
+            loadIssue: async () => {
+                const issue = await api.errorTracking.getIssue(props.id, props.fingerprint)
+                return issue
+            },
+        },
+        properties: {
+            loadProperties: async ({ issue }) => {
+                const firstSeen = dayjs(issue.first_seen)
+                const response = await api.query(
+                    errorTrackingIssueQuery({
+                        issueId: props.id,
+                        dateRange: {
+                            date_from: firstSeen.startOf('minute').toISOString(),
+                            date_to: firstSeen.endOf('minute').toISOString(),
+                        },
+                        volumeResolution: 0,
```

Comment:
Same as before, we should initialise this inside the function. Can we make it optional and not compute the sparkline if we don't need it?